### PR TITLE
Correct description of node drop behavior per issue #4283

### PR DESCRIPTION
**Pull request by duane-r**

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -3612,7 +3612,7 @@ Definition tables
             max_items = 1,  -- Maximum number of items to drop.
             items = { -- Choose max_items randomly from this list.
                 {
-                    items = {"foo:bar", "baz:frob"},  -- Choose one item randomly from this list.
+                    items = {"foo:bar", "baz:frob"},  -- Items to drop.
                     rarity = 1,  -- Probability of getting is 1 / rarity.
```
Comment (Rogier-5) [inline doc/lua_api.txt]: I'd suggest changing 'getting' to 'dropping' as well for consistency in terminology ?

Comment (paramat) [inline doc/lua_api.txt]: Yes please.

**Comment by paramat** [comment]
+1 once changed.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
d6865c4d8d0eb7aaf2d35447ab01e06ef8249010

**Pull request closed without merging**